src/libs/actions/Search.ts

```diff
@@ -156,8 +156,42 @@ function clearAdvancedFilters() {
     Onyx.merge(ONYXKEYS.FORMS.SEARCH_ADVANCED_FILTERS_FORM, values);
 }
 
+function setShouldShowSavedSearchRenameTooltip() {
+    const parameters: SetNameValuePairParams = {
+        name: ONYXKEYS.NVP_SHOULD_SHOW_SAVED_SEARCH_RENAME_TOOLTIP,
+        value: true,
+    };
+
+    const optimisticData: OnyxUpdate[] = [
+        {
+            onyxMethod: Onyx.METHOD.MERGE,
+            key: ONYXKEYS.NVP_SHOULD_SHOW_SAVED_SEARCH_RENAME_TOOLTIP,
+            value: true,
+        },
+    ];
+
+    API.write(WRITE_COMMANDS.SET_NAME_VALUE_PAIR, parameters, {
+        optimisticData,
+    });
```

Comment:
Hmm this is still writing to the API. Can we just keep this as a local value? Like we do for the GBR tooltip? https://github.com/Expensify/App/blob/77e9333655b18e93f0aaf486f06b28b23ddeb11b/src/libs/actions/User.ts#L1322 or `SHOULD_SHOW_COMPOSE_INPUT`